### DISCUSSION TITLE
Can we add multiple organizations to one dataset?

### DISCUSSION THREAD

**Discussion by Suryakandukuri** [ckan/ckan#6913] [General]
I wish to have datasets that would be part of two or more organizations. For example a dataset from finance department related to forests can be part of both forests and finance organizations. I have tried to search for similar set up done by anyone, but in vain. Any help is appreciated. Thanks a lot.

**Comment by pdekraker-epa** [comment]
If you just want to enable write access  you can use the collaborators to add individual collaborators. To add collaborator organizations there is [ckanext-collaborator_orgs](https://github.com/USEPA/ckanext-collaborator_orgs), and I am hopeful this will be incorporated into ckan directly via PR #6905.

If your goal is to actually have two or more organizations displayed as owners and have datasets browsable / searchable via each I do not know of a good way to accomplish this. CKAN is very extensible, so it may be possible to handle this in an extension, but it would be a significant undertaking.

**Reply by Suryakandukuri** [reply]
Thanks, I wish to achieve this.

>  two or more organizations displayed as owners and have datasets browsable/searchable via each.
